controller/system_rollout_controller.go

```diff
@@ -1570,6 +1571,20 @@ func (c *SystemRolloutController) restoreServiceAccounts() (err error) {
 	return nil
 }
 
+var adjustableSettings = [...]string{
+	string(types.SettingNameConcurrentBackupRestorePerNodeLimit),
+	string(types.SettingNameConcurrentReplicaRebuildPerNodeLimit),
+}
+
+func isAdjustableSetting(name string) bool {
```

Comment:
`Adjustable` is a bit of confusing. Any better naming for it?

Reply:
`isIgnoredSetting` better? WDYT?

Reply:
Yeah, sounds good. Thank you.